map/views.py

```diff
@@ -1289,6 +1289,58 @@ def get_tile(self, req, ver, zoom, x, y,
     
         if get_images:
             return rimgs
+
+        if "lslga" in req.GET:
+
+            from PIL import Image, ImageDraw
+            img = Image.open(tilefn)
+
+            ra, dec = wcs.crval
+            pixscale = np.abs(wcs.cd[0]) * 3600
+
+            width, height = img.size
+
+            ralo = ra - ((width / 2) * pixscale / 3600)
+            rahi = ra + ((width / 2) * pixscale / 3600)
+            declo = dec - ((height / 2) * pixscale / 3600)
```

Comment:
This is not quite right -- since the RA coordinate is like longitude, lines of RA converge toward the poles, so you need to divide the dRA by a factor of np.cos(np.deg2rad(dec)).  This is approximate (correct for the center of the image), but good enough for our purposes here.

Reply:
OK, I've added a commit that should correct that.